zrml/authorized/src/lib.rs

```diff
@@ -173,14 +151,9 @@ mod pallet {
     /// Reported outcomes of accounts that are linked through
     /// `MarketDisputeMechanism::Authorized(..)`.
     #[pallet::storage]
-    pub type Outcomes<T: Config> = StorageDoubleMap<
-        _,
-        Blake2_128Concat,
-        MarketIdOf<T>,
-        Blake2_128Concat,
-        T::AccountId,
-        OutcomeReport,
-    >;
+    #[pallet::getter(fn outcomes)]
+    pub type Outcomes<T: Config> =
+        StorageMap<_, Blake2_128Concat, MarketIdOf<T>, Option<OutcomeReport>, ValueQuery>;
```

Comment:
We can use [`OptionQuery`](https://paritytech.github.io/substrate/master/frame_support/storage/types/struct.OptionQuery.html) here.